<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>600</height>
    </size>
   </property>
   <widget class="QPushButton" name="processor">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>150</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Процессор</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="info_programm">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>150</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>О программе</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>150</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ram">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>150</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Модули памяти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mother">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>150</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Материнская плата</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hard">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>150</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Жесткий диск</string>
    </property>
   </widget>
   <widget class="QPushButton" name="os">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>150</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>ОС</string>
    </property>
   </widget>
   <widget class="QLabel" name="switch_info">
    <property name="geometry">
     <rect>
      <x>609</x>
      <y>70</y>
      <width>631</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Информация о процессоре</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>750</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="mode_list">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="apply">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Применить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reboot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="check_update">
    <property name="geometry">
     <rect>
      <x>1054</x>
      <y>550</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Проверить обновления</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
